[2025-04-22T21:48:40.140+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.db_view_operations.create_map_data_view scheduled__2025-04-22T20:00:00+00:00 [queued]>
[2025-04-22T21:48:40.145+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.db_view_operations.create_map_data_view scheduled__2025-04-22T20:00:00+00:00 [queued]>
[2025-04-22T21:48:40.146+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-22T21:48:40.154+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): db_view_operations.create_map_data_view> on 2025-04-22 20:00:00+00:00
[2025-04-22T21:48:40.157+0000] {standard_task_runner.py:57} INFO - Started process 302 to run task
[2025-04-22T21:48:40.158+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data_pipeline', 'db_view_operations.create_map_data_view', 'scheduled__2025-04-22T20:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpjaa3fg00']
[2025-04-22T21:48:40.159+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask db_view_operations.create_map_data_view
[2025-04-22T21:48:40.190+0000] {task_command.py:415} INFO - Running <TaskInstance: weather_data_pipeline.db_view_operations.create_map_data_view scheduled__2025-04-22T20:00:00+00:00 [running]> on host 7883cdd07699
[2025-04-22T21:48:40.240+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='db_view_operations.create_map_data_view' AIRFLOW_CTX_EXECUTION_DATE='2025-04-22T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-22T20:00:00+00:00'
[2025-04-22T21:48:40.249+0000] {logging_mixin.py:151} INFO - Successfully executed SQL from /opt/airflow/src/DBview/map_data_view.sql
[2025-04-22T21:48:40.250+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-22T21:48:40.258+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=db_view_operations.create_map_data_view, execution_date=20250422T200000, start_date=20250422T214840, end_date=20250422T214840
[2025-04-22T21:48:40.292+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-22T21:48:40.306+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
